plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/PhoenixQueryRunner.java

```diff
@@ -49,6 +50,12 @@ private PhoenixQueryRunner()
 
     public static DistributedQueryRunner createPhoenixQueryRunner(TestingPhoenixServer server, Map<String, String> extraProperties)
             throws Exception
+    {
```

Comment:
can we remove this overload? how many places?

Reply:
removed